<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C5ChangeDocInputPrice</class>
 <widget class="QDialog" name="C5ChangeDocInputPrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change input price</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>UUID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="C5LineEdit" name="leUuid">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Goods</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="C5LineEditWithSelector" name="leGoods">
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="C5LineEditWithSelector" name="leGoodsName">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="C5LineEditWithSelector" name="lePrice">
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnChange">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
